Tasks/Kubernetes/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -4,11 +4,13 @@
   "loc.description": "Deploy, configure, update your Kubernetes cluster in Azure Container Service by running kubectl commands.",
   "loc.instanceNameFormat": "kubectl $(command)",
   "loc.group.displayName.commands": "Commands",
-  "loc.group.displayName.registry": "Container Registry Details",
+  "loc.group.displayName.secrets": "Secrets",
   "loc.group.displayName.advanced": "Advanced",
   "loc.group.displayName.output": "Output",
   "loc.input.label.kubernetesServiceEndpoint": "Kubernetes Service Connection",
   "loc.input.help.kubernetesServiceEndpoint": "Select a Kubernetes service connection.",
+  "loc.input.label.namespace": "Namespace",
```

Comment:
Why this change?

Reply:
As discussed with Atul ,the commands and secrets section use the same namespace field so it would be more readable if the namespace field is moved just below the kubernetes service connection field.